ts/packages/ts/src/commands/EvalCommand.ts

```diff
@@ -60,37 +60,42 @@ export class EvalCommand extends Command<EvalCommandReturn> {
   private async evalFromStoreSet(result: EvalCommandReturn): Promise<EvalCommandReturn> {
     const { store, set } = this.parameters;
 
-    const storeQueries = _(result)
-      .values()
-      .flatMap((current) => {
-        const { key } = current.context;
-        return set.hierarchy.map((node) => ({ set: node, key }));
-      })
-      .value();
-    const storeConfigsArray = await store.get(storeQueries);
-    const storeConfigsDict = _(storeConfigsArray)
-      .orderBy([(config) => set.hierarchy.indexOf(config.set)], ['desc'])
-      .uniqBy((config) => config.key)
-      .keyBy((config) => config.key)
-      .value();
-
-    return _.mapValues(result, (current) => {
-      const { context } = current;
-      const storeConfig = storeConfigsDict[context.key];
-
-      if (!storeConfig) {
-        return current;
-      }
+    try {
+      const storeQueries = _(result)
+        .values()
+        .flatMap((current) => {
+          const { key } = current.context;
+          return set.hierarchy.map((node) => ({ set: node, key }));
+        })
+        .value();
+
+      const storeConfigsArray = await store.get(storeQueries);
+      const storeConfigsDict = _(storeConfigsArray)
+        .orderBy([(config) => set.hierarchy.indexOf(config.set)], ['desc'])
+        .uniqBy((config) => config.key)
+        .keyBy((config) => config.key)
+        .value();
+
+      return _.mapValues(result, (current) => {
+        const { context } = current;
+        const storeConfig = storeConfigsDict[context.key];
+
+        if (!storeConfig) {
+          return current;
+        }
 
-      return {
-        ...current,
-        result: {
-          origin: EvaluatedConfigOrigin.StoreSet,
-          source: `parameters.store=${context.store},parameters.set=${context.set}`,
-          value: storeConfig.value,
-        },
-      };
-    });
+        return {
+          ...current,
+          result: {
+            origin: EvaluatedConfigOrigin.StoreSet,
+            source: `parameters.store=${context.store},parameters.set=${context.set}`,
+            value: storeConfig.value,
+          },
+        };
+      });
+    } catch (error) {
+      throw new ConfigStoreError('failed to get store configs', error.message, [['EvalCommand', 'evalFromStoreSet']]);
```

Comment:
I'm unsure why everything is wrapped in a try-catch instead of `store.get` exclusively. Please explain the rationale.